features/git-town/town.feature

```diff
@@ -1,8 +1,8 @@
 Feature: Show correct git town usage
 
   Scenario: invalid git town command
-    When I run `git town invalidcommand`
-    Then I see
+    When I run `git town invalidcommand` while allowing errors
+    Then I get the error
```

Comment:
`Then I see` was verifying the output matched exactly instead of just being included. 
I'd prefer it to stay that way. I'm about to start a PR that changes
```
When I run `<cmd>` while allowing errors
```
to 
```
When I run `<cmd>` it errors
```
which will also verify that the command errors instead of allowing it